react-native-libraries.json

```
@@ -11581,6 +11581,28 @@
     "web": true,
     "expoGo": true
   },
+  {
+    "githubUrl": "https://github.com/Aris-ngoy/react-native-irano",
+    "npmPkg": "react-native-irano",
+    "examples": [
+      "https://github.com/Aris-ngoy/react-native-irano/tree/main/example"
+    ],
+    "images": [
+      "https://github.com/Aris-ngoy/react-native-irano/blob/main/assets/1.png?raw=true",
+      "https://github.com/Aris-ngoy/react-native-irano/blob/main/assets/2.png?raw=true",
+      "https://github.com/Aris-ngoy/react-native-irano/blob/main/assets/3.png?raw=true",
+      "https://github.com/Aris-ngoy/react-native-irano/blob/main/assets/4.png?raw=true"
```

```suggestion
      "https://raw.githubusercontent.com/Aris-ngoy/react-native-irano/refs/heads/main/assets/1.png",
      "https://raw.githubusercontent.com/Aris-ngoy/react-native-irano/refs/heads/main/assets/2.png",
      "https://raw.githubusercontent.com/Aris-ngoy/react-native-irano/refs/heads/main/assets/3.png",
      "https://raw.githubusercontent.com/Aris-ngoy/react-native-irano/refs/heads/main/assets/4.png"
```

Nit, let's use direct raw URLs to avoid redirects.